ansible/roles/baselayout/tasks/partials/repo/centos5.yml

```diff
@@ -14,5 +14,5 @@
 
 - name: install epel
   yum:
-    name: "http://dl.fedoraproject.org/pub/epel/5/{{ ansible_architecture }}/epel-release-5-4.noarch.rpm"
+    name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-5.noarch.rpm"
```

Comment:
hah, you're right, I got that URL off the EPEL main page and it's broken there too (their 6 and 7 links work tho), fixed in latest commit